packages/govern-server/src/api-graphql/resolvers.ts

```diff
@@ -20,12 +20,22 @@ export default function resolvers(govern: GovernCore): IResolvers {
     Dao: {
       async registryEntries(parent) {
         return Promise.all(
-          parent.registryEntries.map(async ({ id }: { id: string }) =>
-            govern.registryEntry(id)
-          )
+          parent.registryEntries
+            .map(async ({ id }: { id: string }) => {
+              const entry = await govern.registryEntry(id)
+              return entry
+                ? {
+                    ...entry,
+                    executor: govern.dao(entry.executor.address),
+                  }
+                : null
+            })
+            .filter(Boolean)
         )
       },
+      async queues(parent) {
```

Comment:
What exactly will this return? A DAO only has one queue but a queue with many container hashes stored.

Reply:
This returns all the queues associated to the DAO via containers.

Reply:
I see.. totally forgot you are able to register multiple DAO workflows 👌